packages/web-config-server/src/utils/logApiRequest.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+export const logApiRequest = (models, apiName, version) => async (req, res, next) => {
```

Comment:
Would the util exported from `server-boilerplate` work here instead?

Reply:
Technically the way the `userId` is determined is different. However if I'm being honest  I was a bit worried about creating a dependency on server-boilerplate. Now that I check, there's already a dependency there! 🤦

I thiiiink it's might still be cleaner to do it separate rather than try and make the `userId` logic more generic... hopefully we switch this bad boy over to server-boilerplate sometime in the not too distant future and then we can scrap this?

Reply:
Sweet sounds good - so long as you'd considered it!